lib/github/index.js

```diff
@@ -17,7 +17,8 @@ module.exports = (robot) => {
 
   robot.on('push', middleware(push));
 
-  robot.on(['pull_request.opened', 'pull_request.closed', 'pull_request.reopened', 'pull_request.edited'], middleware(pullRequest));
+  // robot.on(['pull_request.opened', 'pull_request.closed', 'pull_request.reopened', 'pull_request.edited'], middleware(pullRequest));
+  robot.on(['pull_request.opened', 'pull_request.closed', 'pull_request.reopened', 'pull_request.edited'], workerMiddleware('pullRequest'));
```

Comment:
In my mind, the ideal here would be to start with one handler (on of our bigger ones) and move over bit by bit to the background queue.